onnxruntime/core/session/inference_session.cc

```diff
@@ -486,7 +486,7 @@ common::Status InferenceSession::SaveToOrtFormat(const std::basic_string<ORTCHAR
   sb.add_model(model);
   sb.add_session_state(session_state);
   auto session = sb.Finish();
-  builder.Finish(session);
+  builder.Finish(session, fbs::InferenceSessionIdentifier());
```

Comment:
One thing I was thinking is we can get rid of this identifier. which is not used anyway currently
If we use buffer verifier, seems the identifier is not really useful anymore